src/components/review/WriteItems/DropdownPrice.tsx

```diff
@@ -0,0 +1,79 @@
+import Image from 'next/image';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+import PriceList from './PriceList';
+
+interface StyledDPProps {
+  idx: number;
+  currentOpen: number;
+  onSetCurrentOpen: (idx: number) => void;
+}
+
+function DropdownPrice(props: StyledDPProps) {
+  const { idx, currentOpen, onSetCurrentOpen } = props;
+  const [selectedPrice, setSelectedPrice] = useState('0');
+  const [isOpen, setIsOpen] = useState(false);
+  const [isSelected, setIsSelected] = useState(false);
+  const toggle = () => setIsOpen((prevIsOpen) => !prevIsOpen);
+
+  const onSelectedPrice = (price: string) => {
+    setSelectedPrice(price);
+    toggle();
+    setIsSelected(true);
+  };
+
+  const handleClick = () => {
+    onSetCurrentOpen(idx);
+    toggle();
+  };
+
+  useEffect(() => {
+    if (idx !== currentOpen && isOpen) {
+      toggle();
+    }
+  }, [currentOpen]);
```

Comment:
```suggestion
  useEffect(() => {
    if (idx !== currentOpen && isOpen) {
      toggle();
    }
  }, [idx, isOpen, currentOpen]);
```

의존성 배열을 수정해주면 좋을 것 같아유